modules/ROOT/pages/fcct-config.adoc

```diff
@@ -0,0 +1,107 @@
+:experimental:
+= FCCT Configuration and Common Use Cases
```

Comment:
We can call this "Provisioning and configuration", see https://coreos.com/os/docs/latest/provisioning.html.